src/twisted/persisted/test/test_styles.py

```diff
@@ -90,6 +102,16 @@ def test_instanceBuildingNameNotPresent(self):
         self.assertIsNot(m, foo.method)
 
 
+    def test_copyFunction(self):
+        """
+        Copying a function should exercise the pickle code
+        and preserve the function name
+        """
+        f = copy.copy(sampleFunction)
+        self.assertEqual(f, sampleFunction)
+        self.assertEqual(f.__name__, sampleFunction.__name__)
```

Comment:
This docstring is misleading.
From what I see in the implementation, it returns the exact same function... so is normal to have the same name.

Reply:
What's the suggestion here?

Reply:
`Copying a function returns the same reference, without creating an actual copy.` . is my suggestion from the diff

Reply:
OK, I incorporated your patch into this PR.